include/crm/common/util.h

```diff
@@ -62,6 +62,9 @@ char *crm_strdup_printf(char const *format, ...) __attribute__ ((__format__ (__p
 int pcmk__parse_ll_range(const char *srcstring, long long *start, long long *end);
 gboolean pcmk__str_in_list(GList *lst, const gchar *s);
 
+bool pcmk_str_any_of(const char *s, ...) G_GNUC_NULL_TERMINATED;
+bool pcmk_str_none_of(const char *s, ...) G_GNUC_NULL_TERMINATED;
```

Comment:
I'd rather these be internal (in include/crm/common/internal.h). As Beekhof once put it to me, "we don't want libcrmcommon to become another glib". If something's not cluster-related or required to use something cluster-related, we should prefer internal to keep the flexibility to change at will.

(Of course a bunch of old functions are public that should be internal by this principle, but we have to wait until the next API compatibility break to do that)

Reply:
That is a good, concise rule to live by.  Changed.